assets/lang/sklmessages_pl_PL.properties

```diff
@@ -84,7 +84,7 @@ editor.compatibility.tooltip=Skiny nie dzia\u0142aj\u0105 w tym trybie, ale niek
 
 settings.title=Ustawienia launchera
 settings.lang=J\u0119zyk
-settings.lang.help=Widzisz jakie\u015b b\u0142\u0119dy w t\u0142umaczeniu?
+settings.lang.help=Widzisz jakie\u015b b\u0142\u0119dy w t\u0142umaczeniu?\u0020
```

Comment:
Is this whitespace character (\u0020) really necessary? You added it also in #34 and @skmedix rejected it without giving a reason, so I don't know it should remain or not.

Reply:
@Greg-21 Yeah, in the first build, labels didn't have any spacing between them, so the workaround for that was adding an extra space. But now it's not needed.

Reply:
@skmedix Oh, OK. I understand now. Thanks for your quick response.

Reply:
@skmedix Oh, OK. I understand now.

Thanks for your invitation. I don't have a Discord account, so at first I have to create one. I don't have a Discord account yet, because, to be honest, I don't like Discord, because its app is built on Electron (which uses Google's Chromium; also you are using Chromium, but this is other thing :) ). I'm sometimes visiting your Discord server (especially "30-commits") using guest account. So maybe in the future I'll join.

Reply:
`also you are using Chromium, but this is other thing`
Im not :P

Reply:
@skmedix My bad. You are using WebView, right? I only know two projects called "WebView". One of them is Oracle's JavaFX WebView and the second one is Google's WebView. So my first thought was that you are using Google's WebView. Because it's extremely popular.